solutions/cf-1225D.mdx

```diff
@@ -0,0 +1,101 @@
+---
+id: cf-1225D
+source: CF
+title: Power Products
+author: Jesse Choe
+---
+
+[Official Editorial](https://codeforces.com/blog/entry/89810)
+
+
```

Comment:
since this code is already well commented, I don't see why converting it to an explanation would hurt

Reply:
Well, the official editorial already has a good explanation lmao.